torch/_torch_docs.py

```diff
@@ -2701,6 +2701,43 @@ def merge_dicts(*dicts):
     input (Tensor): the PyTorch tensor to test
 """)
 
+add_docstr(torch.is_nonzero,
+           r"""
+is_nonzero(input) -> (bool)
+
+Returns True if the :attr:`input` is a scalar which is not equal to zero
+after type conversions.
+i.e. not equal to ``torch.Tensor([0.])`` or ``torch.Tensor([0])`` or
```

Comment:
torch.Tensor is old-fashioned, use torch.tensor (all lowercase)

Reply:
Updated